client/src/views/Models/Model.vue

```diff
@@ -108,6 +85,7 @@
   import { GraphInterface, GraphLayoutInterface, GraphNodeInterface, SubgraphInterface, GraphLayoutInterfaceType } from '@/types/typesGraphs';
   import * as Model from '@/types/typesModel';
   import { CosmosSearchInterface } from '@/types/typesCosmos';
+  import { GrometTypesEnum } from '@/types/typesGroMEt';
```

Comment:
This is not an big issue, but I try to keep GroMET as much as possible out of the HMI.
This enum would be better place in `typesModel.ts` as `GraphType` because we also use those values there:
```JS
/* Define a graph (nodes and edges) per type of a model. */
type Graph = {
  file: string,
  type: 'PetriNetClassic' | 'FunctionNetwork',
  metadata?: Array<GraphMetadata>,
  graph: Graph.GraphInterface,
};
```